pkg/cmd/roachtest/cluster.go

```diff
@@ -3049,3 +3049,41 @@ func (c *clusterImpl) GetPreemptedVMs(
 	}
 	return allPreemptedVMs, nil
 }
+
+// GetHostErrorVMs gets any VMs that were part of the cluster but has a host error.
+func (c *clusterImpl) GetHostErrorVMs(
+	ctx context.Context, l *logger.Logger,
+) ([]vm.PreemptedVM, error) {
+	if c.IsLocal() || !c.spec.UseSpotVMs {
```

Comment:
We don't want the `!c.spec.UseSpotVMs` here.

Reply:
Removed